assets/js/components/ControlPanel.tsx

```diff
@@ -36,6 +36,7 @@ import {
   SourceType,
 } from "../utils/rtcUtils";
 import { Mode } from "./StreamArea";
+import { getFontColor } from "./ParticipantsList";
```

Comment:
It looks a little bit strange that you import `getFontColor` from `ParticipantsList`, maybe we could create some utils file for this?

Reply:
Fixed.